src/app/auth/components/login.component.html

```diff
@@ -13,11 +13,11 @@
       <form novalidate [formGroup]="authForm" (ngSubmit)="login()">
         <div>
           <md-input-container>
-            <input md-input formControlName="username" placeholder="User name" required type="text" focused>
+            <input md-input formControlName="username" placeholder="User name" required type="text" autocomplete="off" focused>
```

Comment:
In my opinion autocomplete for username should be enabled by default, except for users who are really paranoid. So I think this should be a config setting that is read during build. Can you investigate how to make that happen?